automation/taskcluster/decision_task.py

```diff
@@ -247,13 +292,33 @@ def create_compare_locales_task():
     )
 
 
+def populate_chain_of_trust_required_but_unused_files():
```

Comment:
Nit: Let's reuse the existing function https://github.com/mozilla-mobile/reference-browser/blob/9d56ab118d4fc6b9bd7e00c4f6f59a9c89c302c4/automation/taskcluster/decision_task_nightly.py#L119

Let's move that function in a file called `chain_of_trust.py` just like we did in Fenix https://github.com/mozilla-mobile/fenix/blob/f9c791c93b64c71328f425e8af192e95f37e70db/automation/taskcluster/lib/chain_of_trust.py#L4